docs/integration-guides/work-generation.md

```diff
@@ -87,19 +87,22 @@ graph TD
 
 ### Work generated without using the node
 
+!!!tip "Lower thresholds for receive blocks"
+    **Receive blocks** benefit from a lower work threshold. In the following guide, replace uses of `network_minimum` and `network_current` with `network_receive_minimum` and `network_receive_current`, respectively, to benefit from the lower threshold.
+
 ``` mermaid
 graph TD
     M{Access to a node?} -->|yes| N[active_difficulty <a href='/commands/rpc-protocol/#active_difficulty'><b>RPC</b></a> or <a href='/integration-guides/websockets/#active-difficulty'><b>WS</b></a>]
     M --> |no| O_1(<a href='/protocol-design/networking/#node-telemetry'><b>Telemetry</b></a>)
-    N -->P_1(Generate work at<br><b>network_minimum</b><br>or <b>network_receive_minimum</b><br>difficulty)
```

Comment:
Did you mean to roll this back? And the other update further down?

Reply:
It seemed a bit redundant to include `network_minimum` in the arrow here and `network_current` below, but could also go back to how it was.

Reply:
👍